hansol/app.js

```diff
@@ -0,0 +1,46 @@
+const http = require('http')
+const express = require('express')
+const morgan = require('morgan')
+const cors = require('cors')
+const dotenv = require('dotenv')
+const { DataSource } = require('typeorm')
+
+dotenv.config()//환경변수를 사용하기 전에 놔야함 밑으로 가면 에러발생.
+
```

Comment:
- mport 순서 컨벤션에 맞게 수정해주세요! 사소한 부분이라 놓치기 쉽지만 처음 코드를 작성할 때부터 신경쓰고 작성하게 되면 추후에 관리하기도 용이하며 가독성도 좋아집니다.
- 다음과 같은 순서로 import 해줍니다.
    - Bulit-in 패키지
    - 3rd-party 패키지
    - custom 패키지
```suggestion
//// Built-in package
const http = require('http')

//// 3rd-party package
require('dotenv').config();
const express = require('express');
const cors    = require('cors');
const morgan  = require('morgan');
const { DataSource } = require('typeorm');

//custom package
const api = require('./../api')

const app = express();
```